tests/unit/CoreImageTest.php

```diff
@@ -36,103 +38,336 @@ public function setUp(): void {
 	public function tearDown(): void {
 		// your tear down methods here
 		wp_delete_post( $this->post_id, true );
-		\WPGraphQL::clear_schema();
 
 		parent::tearDown();
+
+		\WPGraphQL::clear_schema();
 	}
 
-	public function test_retrieve_core_image_media_details() {
-		$query  = '
+	public function query() {
+		return '
 			fragment CoreImageBlockFragment on CoreImage {
 				attributes {
 					id
-				}
-				mediaDetails {
-					height
 					width
+					height
+					alt
+					align
+					src
+					style
+					sizeSlug
+					# lightbox # not supported yet
+					# aspectRatio # not supported yet
+					# scale # not supported yet
```

Comment:
You should target the version(s?) where it _does_ work in a separate test and be more specific in the comment

![image](https://github.com/user-attachments/assets/2846714a-c65e-4b05-b58e-37288e2ff353)